### ISSUE TITLE
Undefined symbol for python-libcombine 0.2.4

### ISSUE THREAD

**Issue by yannikschaelte** [sbmlteam/libCombine#29]
Using python-libcombine 0.2.4 in [PEtab](https://github.com/petab-dev/petab) (python 3.6-3.8, ubuntu), I get

```
ImportError: /opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages/libcombine/_libcombine.cpython-38-x86_64-linux-gnu.so: undefined symbol: deflateInit2_
```

The full error trace is here: https://github.com/PEtab-dev/PEtab/pull/435/checks?check_run_id=660417473. It worked for 0.2.3.

**Comment by yannikschaelte** [comment]
(basically same as #26, I just remember)

**Comment by fbergmann** [comment]
thanks for letting me know ... i'll look into it

**Comment by fbergmann** [comment]
it took a while but i think i have it now: 

<https://pypi.org/project/python-libcombine/0.2.6>

I'll generate the windows / mac binaries next, but linux at least should work.

**Comment by yannikschaelte** [comment]
thanks for the fast fix, it works with 0.2.6 (on linux; that's all I can check).
